pom.xml

```diff
@@ -14,18 +14,14 @@
   <packaging>hpi</packaging>
 
   <properties>
-<!--    <jenkins.version>2.190.3</jenkins.version>-->
-    <jenkins.version>2.277.4</jenkins.version>
-    <java.level>8</java.level>
```

Comment:
java level is now defined by the version of jenkins being targeted [ref](https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.40)